### PR TITLE
curs-01: Fix small typos

### PR DESCRIPTION
**Pull request by alexghergh**
This commit fixes small typos. More of a visual change than anything significant.

### CODE REVIEW

File: curs/curs-01-prog/demo/inspect_array.c [hunk 1]
```diff
@@ -5,7 +5,7 @@ int main(void)
 	int key[256];
```
Comment (dbaluta) [inline curs/curs-01-prog/demo/inspect_array.c:6]: @alexghergh Please add 'curs-01' prefix to the subject. 

Like this ```curs-01: Fix small typos```

**Comment by RazvanN7** [comment]
Please rebase and we're good to go.

**Comment by RazvanN7** [comment]
@alexghergh please rebase.

**Comment by RazvanN7** [comment]
Ah nvm, this has been fixed.

**Pull request closed without merging**